src/v3/src/components/CaptchaContainer/CaptchaContainer.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import HCaptcha from '@hcaptcha/react-hcaptcha';
+import { Box } from '@okta/odyssey-react-mui';
+import { useEffect, useRef } from 'preact/hooks';
+import ReCAPTCHA from 'react-google-recaptcha';
+
+import { useWidgetContext } from '../../contexts';
+import { useOnSubmit } from '../../hooks';
+import {
+  CaptchaContainerElement,
+  UISchemaElementComponent,
+} from '../../types';
+
+const CaptchaContainer: UISchemaElementComponent<{
+  uischema: CaptchaContainerElement
+}> = ({ uischema }) => {
+  const {
+    options: {
+      siteKey,
+      type: captchaType,
+      captchaId,
+    },
+  } = uischema;
+
+  const { dataSchemaRef } = useWidgetContext();
+  const onSubmitHandler = useOnSubmit();
+  const dataSchema = dataSchemaRef.current!;
+  const captchaRef = useRef<ReCAPTCHA | HCaptcha>(null);
+
+  useEffect(() => {
+    // set the reference in dataSchema context to this captcha instance
+    dataSchema.captchaRef = captchaRef;
+    //  For some reason Hcaptcha does not initially auto render only when in development mode
+    //  This delayed manual render is only used in development mode for hcaptcha
+    if (process.env.NODE_ENV === 'development' && captchaType === 'HCAPTCHA') {
```

Comment:
could possibly be an issue with compiling typescript and hot module updating,
https://github.com/hCaptcha/react-hcaptcha/issues/53#issuecomment-707293639

Reply:
Can you test this by running without the watch mode on? That should disable hot module updating, I think.

Reply:
Turning off hot module updating didnt seem to fix it.  I added a link to a github issue in the comment